src/node/node.ino

```diff
@@ -815,6 +823,10 @@ void i2cTransmit()
             ioBufferWrite32(millis());
             break;
 
+        case READ_VOLTAGE: // RF: READ_VOLTAGE will get the current node voltage from the defined analog pin
+            ioBufferWrite32(voltageRead()); 
```

Comment:
Not sure which ioBufferWrite function is needed.

Reply:
Analog reads are 10-bit, so the raw data would fit best in the 16-bit write.

```suggestion
            ioBufferWrite16(voltageRead()); 
```